tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -2867,3 +2868,14 @@ def test_get_orderby_clauses():
         assert "value IS NULL" in select_clause[0]
         # test that clause name is in parsed
         assert "clause_1" in parsed[0]
+
+
+def test_get_metric_history_on_non_existent_metric_key():
+    store = SqlAlchemyStore("sqlite:///:memory:", ARTIFACT_URI)
```

Comment:
You'll want to use the `experiment_factory` to create a new experiment to prevent collisions in other suites :)